grand/sim/detector/antenna_model.py

```diff
@@ -109,6 +109,26 @@ def __init__(self, du_type="GP300"):
             path_ant = grand_add_path_data("detector/Light_GP300Antenna_Zarm_leff.npz")
             self.leff_z = tabulated_antenna_model(path_ant)
             
+        elif du_type=="GP300_nec":
```

Comment:
I added some explanatory comments to the script but mainly I provided the link where one can find all the available documents (and presentations)  explaining how these simulations were created (input parameters etc) as well as a comparison of results between different models (hiss, nec Matlab).

Reply:
thanks!. The links are a good addition. However, i was hoping to get some documentation in the code as documentation for the class AntennaModel, in particular what are the available du_type options and what do they imply. Also, it would be usefull for the messages ""Loading GP300 antenna model ..." would be different for each model (and present in the case of the "Horizon" du_type) so that any missconfiguration can be detected at run time.